devito/passes/clusters/blocking.py

```diff
@@ -24,28 +56,12 @@ def __init__(self, options):
     def _make_key_hook(self, cluster, level):
         return (tuple(cluster.guards.get(i.dim) for i in cluster.itintervals[:level]),)
 
-    @timed_pass(name='blocking')
     def process(self, clusters):
-        # Preprocess: heuristic: drop TILABLE from innermost Dimensions to
-        # maximize vectorization
-        processed = []
-        for c in clusters:
-            ntilable = len([i for i in c.properties.values() if TILABLE in i])
-            ntilable -= int(not self.inner)
-            if ntilable <= 1:
-                properties = {k: v - {TILABLE} for k, v in c.properties.items()}
-                processed.append(c.rebuild(properties=properties))
-            elif not self.inner:
-                d = c.itintervals[-1].dim
-                properties = dict(c.properties)
-                properties[d] = properties[d] - {TILABLE}
-                processed.append(c.rebuild(properties=properties))
-            else:
-                processed.append(c)
-
-        processed = super(Blocking, self).process(processed)
-
-        return processed
+        if self.levels < 1:
+            return clusters
+        else:
+            processed = super(Blocking, self).process(clusters)
```

Comment:
`return super(...)` (because it's a one-liner scope)

Reply:
ok